Plugin/SetTaxClassId.php

```diff
@@ -106,11 +106,12 @@ public function createQuery(string $taxIdColumn, string $tableName): string
     /**
      * Add the switch case to the query.
      */
-    public function addCase(string $query, string $taxIdColumn): string
+    public function addCase(string $query, string $taxIdColumn): ?string
     {
         if (!($mappings = $this->scopeConfig->getValue('akeneo_connector/product/tax_id_mapping'))) {
-            return;
+            return null;
```

Comment:
Als ik de code doorlees en hoe het gebruikt wordt gok ik dat we hier 
`return $query;` willen doen zodat onze query nog uitgevoerd wordt toch?

https://github.com/justbetter/magento2-akeneo-bundle/blob/master/Plugin/SetTaxClassId.php#L135
https://github.com/justbetter/magento2-akeneo-bundle/blob/master/Plugin/SetTaxClassId.php#L109